features/add_ressource_to_item.feature

```diff
@@ -0,0 +1,19 @@
+#language: fr
+
+Fonctionnalité: Ajouter une ressource en tant que fichier 
+
+
+Scénario: Ajouter une ressource en tant que fichier 
```

Comment:
The name of a scenario should correspond to a special case of the feature.
Duplicating the feature name just means that you should let the scenario title empty.